torchprime/data/__init__.py

```diff
@@ -1,3 +1,11 @@
 """
 The data module contains datasets and efficient dataloading implementations.
 """
+
+from .dataset import make_huggingface_dataset
```

Comment:
Rename this to `make_basic_dataset` or `make_train_dataset` to contrast with `make_sft_dataset`?